### DISCUSSION TITLE
Stream from Android to Snapserver

### DISCUSSION THREAD

**Discussion by igbjnI05bF** [snapcast/snapcast#1271] [General]
When my Android phone is connected to my local LAN, which has a snapcast server on it, is there any way to set the snapcast server as the default audio source to stream all sounds from my phone to snapcast? I'm looking for something similar to what apple has with airplay. In the same way that you can send all android audio to bluetooth earbuds, I want to do that but to snapcast, so *all* sounds go to it.

Imagine how great it would be to be outside listening to a podcast with your earbuds, then walk inside your house and change the output from bluetooth to your snapcast server and continue listening to your podcast, but now with your whole-home audio. That's my dream. How do I make this happen lol

**Comment by Cris--** [comment]
To grab all audio from an Android phone you need root and the AirAudio app! An AirAudio Magisk module is also available.

Or something like a single audio player app with in-built Snapcast with broadcast feature to play files/streams. I don't know any app like this. Anyone?

**Reply by Cris--** [reply]
Found the Snap.Net app with a Snapcast client/broadcast feature for Android.

[https://github.com/stijnvdb88/Snap.Net](url)


Now dealing with no playing audio in Kodi or casting to clients if i broadcast from Android phone to the Rpi5/LibreElec 13 (Kodi)/Snapserver addon (Addon v 12.80.1.0; Snapcast Server 0.28.0).

If i play audio with Kodi all Snapcast clients in sync (including Kodi audio playback (Snapclient Addon).

**Comment by rnestler** [comment]
With the [AV Capture API](https://developer.android.com/media/platform/av-capture) from Android it should be possible the grab Audio from other apps and stream it to the snapcast server.
